flask_restplus/swagger.py

```diff
@@ -377,6 +377,8 @@ def serialize_resource(self, ns, resource, url, route_doc=None, **kwargs):
         doc = self.extract_resource_doc(resource, url, route_doc=route_doc)
         if doc is False:
             return
+        for _n,_v in ns.models.items():
```

Comment:
This would include all models. We might want to consider adding a switch for users to have either the new functionality or the old functionality.